### PR TITLE
Increase maximum for zoom plugin

### PR DESCRIPTION
**Pull request by meisenzahl**
Closes https://github.com/elementary/gala/issues/876

### CODE REVIEW

**Comment by meisenzahl** [comment]
@mmohsinkhan has described that he needs at least a zoom factor of `6.0f`. Should the maximum then be even higher, e.g. `10.0f`? Should the step size still be `0.5f`?

**Comment by mmohsinkhan** [comment]
Max magnification factor 2.5f set currently is not enough for people like me. I need to magnify desktop more than 2.5 times to make it usable for me.
I don't think setting max factor to 10f would do any harm, on the bright people like me would be able to use Elementary out of the box.

Thanks

File: plugins/zoom/Main.vala [hunk 1]
```diff
@@ -82,7 +82,7 @@ namespace Gala.Plugins.Zoom {
             // Nothing to do if zooming out of our bounds is requested
             if (current_zoom <= 1.0f && !@in)
                 return;
-            else if (current_zoom >= 2.5f && @in)
+            else if (current_zoom >= 10.0f && @in)
                 return;
```
Comment (danrabbit) [inline plugins/zoom/Main.vala:86]: While we're here, a little bit of feedback couldn't hurt :)
```suggestion
                Gdk.beep ();
                return;
```

Comment (meisenzahl) [reply plugins/zoom/Main.vala:87]: Many thanks for the feedback :) It is your love for detail that makes elementary OS so unique.

**Comment by meisenzahl** [comment]
The solution was much easier than I thought #Bitesize

**Comment by mmohsinkhan** [comment]
Thanks guys for the support.

**Pull request merged**